test/e2e/atlas/search_test.go

```diff
@@ -223,4 +223,173 @@ func TestSearch(t *testing.T) {
 		expected := fmt.Sprintf("Index '%s' deleted\n", indexID)
 		assert.Equal(t, string(resp), expected)
 	})
+
+	t.Run("Create combinedMapping", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			"loadSampleData",
+			g.clusterName,
+			"--projectId", g.projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		_, err := cmd.CombinedOutput()
+		require.NoError(t, err)
```

Comment:
this is a nice addition, did you add it because you were getting errors? it may make sense to make this a precondition of the whole test suite, also I created [CLOUDP-171360](https://jira.mongodb.org/browse/CLOUDP-171360) as I'm realizing we don't have a way to check we are doing loading also why I think it makes sense to move this earlier to give ti time to finish

Reply:
Oh no, I added this because we were creating index on dummy db/collection/fields. And in order to ensure index creation using static and dynamic mapping works, I needed a proper field (preferably document type).

I will modify this as a precondition. Thanks for the review comments :)